src/Components/ImportContactsDialog/ImportContactsDialog.jsx

```diff
@@ -109,9 +109,12 @@ const ImportContactsDialog = ({ status, callback, onImport }) => {
 
             var all = document.getElementsByClassName("initial__modal");
             for (var i = 0; i < all.length; i++) {
-              all[i].style.display = "block";
+              all[i].style.display = "none";
             }
 
+            // Mimicing close button click
+            document.getElementsByClassName("btn-close")[0].click();
```

Comment:
Please use state or reducer to close the dialog box instead of this

Reply:
@biswanathsahoo Updated it